app/src/client/client.ts

```diff
@@ -102,4 +102,10 @@ export class NotdClient extends ServiceClient {
     const request = new Endpoints.SubscribeRequest(email);
     await this.makeRequest(method, path, request, Endpoints.SubscribeResponse);
   };
+  public retrieveCollectionUpdate = async (address: string): Promise<void> => {
```

Comment:
newline between functions

Reply:
also this function should be called `updateCollection`, you arent retrieving anything here!

Reply:
update the request and response class names too pls